CoreScriptsRoot/LoadingScript.lua

```diff
@@ -762,3 +763,24 @@ game:GetService("ReplicatedFirst").RemoveDefaultLoadingGuiSignal:connect(handleR
 if game:GetService("ReplicatedFirst"):IsDefaultLoadingGuiRemoved() then
 	handleRemoveDefaultLoadingGui()
 end
+
+local UserInputServiceChangedConn;
```

Comment:
The reason to use a `;` at the end of the line is to prevent mistakes such as this:

local something


= "blagh"

In the above example something would be accidentally initialized to "blagh"